qiita_db/study.py

```diff
@@ -347,6 +348,56 @@ def create(cls, owner, title, efo, info, investigation=None):
 
         return cls(study_id)
 
+    @classmethod
+    def delete(cls, id_):
+        r"""Deletes the study from the database
+
+        Parameters
+        ----------
+        id_ : integer
+            The object identifier
+
+        Raises
+        ------
+        QiitaDBError
+            If the sample_(id_) table exists means a sample template exists
+        """
+        cls._check_subclass()
+
+        conn_handler = SQLConnectionHandler()
+        if exists_table('sample_%d' % id_, conn_handler):
```

Comment:
A question about the future. Is a user going to be able to add raw data, preprocessed data or processed data (otu tables) w/o a sample template? If so, we should check if the study has something else...